spec/helpers.lua

```diff
@@ -3021,6 +3021,7 @@ local function get_clustering_protocols()
   local confs = {
     wrpc = "spec/fixtures/custom_nginx.template",
     json = "/tmp/custom_nginx_no_wrpc.template",
```

Comment:
does this still need a custom template? can we use the #9071 instead?

Reply:
If we rely on this switch to implement the test, then it would be difficult to remove this switch.

Reply:
that's fine, this will become a test for the switch itself, since we don't have such test. and we will know what will break/change when the test is removed.

Reply:
The "json" part is to make sure 3.0 CP works with 2.8 DPs, and they will not be removed when the switch is removed.